.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+- '10'
+cache:
+  directories:
+  - node_modules
+jobs:
+  include:
+  - stage: tests
+    script:
+    - npm run test
+  - stage: Publish
+    if: "(NOT type IN (pull_request)) AND (branch = master)"
+    script:
+      - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
+      - npm run publish
```

Comment:
Might want to run a version step before that? Or shall we run that before pushing?